tests/examples/segs.py

```diff
@@ -263,6 +294,37 @@ def undersegmentation_3d() -> tuple[np.ndarray, np.ndarray]:
     return gt, pred
 
 
+def no_overlap_3d() -> tuple[np.ndarray, np.ndarray]:
```

Comment:
Maybe we want to declare test fixtures for these canonical examples in conftest.py? Makes them easier to use just by declaring them as arguments, and saves us an import.

Reply:
We had them as fixtures and then ran into issues with calling the functions outside the tests to make visualizations put in the docs, and we didn't want to duplicate the code. Plus for the graph examples we want to parameterize over the functions, and having some be fixtures and some not seemed inconsistent.

Reply:
Ahh got it, thanks for the additional context